labelbox/schema/role.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass
+
+from labelbox.orm.model import Field
+from labelbox.orm.db_object import DbObject
+from labelbox.schema.project import Project
+
+_ROLES = None
+
+
+def get_roles(client):
```

Comment:
🥳 
I might recommend  typing annotations on `ROLES` or `get_roles` but not a big deal

Reply:
fixed.